inc/class-statify-frontend.php

```diff
@@ -116,6 +116,20 @@ public static function track_visit() {
 		return self::_jump_out( $is_snippet );
 	}
 
+	/**
+	 * Track the page view via AJAX.
+	 *
+	 * @return void
```

Comment:
In my mind there is not helpful. Also the hint on phpdoc - This tag should not be used for constructors or methods defined with a void return type.

Reply:
We're using this tag in 15 places... Apparently all introduced by myself (default pattern in my IDE because it's mandatory in most of my styleguides - and btw. the PHPdoc ref marks this as "MAY be omitted").

We shouldn't mix it up, so either remove all or none.
Along with some int(eger) and bool(ean) tags probably little out of scope of this PR.

Reply:
Yes, plugincollektiv need a style guide, simplest way is the WP codex. In this context it is only a hint and the result of different views, works outside of the pluginkollektiv projects. Feel free to leave that or remove all ;)